src/styles/blocks/card-container.scss

```diff
@@ -0,0 +1,16 @@
+.cards-container {
+  box-sizing: border-box;
```

Comment:
it should be in `*` selector

Reply:
1) before first commit i run npm run lint and npm test successfully 
2) now I did all proposed remarks, run again npm tests and all of them are passed locally but now I can see strange behavior  - when I run "npm start" I can see  correct appearance of my catalog, but if I push "go live"  I see my page like without styles and if run again "npm run lint" I can see following errors 
src/styles/main.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/blocks/card-container.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/blocks/card.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/blocks/header.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/blocks/nav.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/blocks/reviews-container.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/blocks/stars.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/utils/_functions.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

src/styles/utils/_variables.scss
 1:1  ✖  Unknown rule no-invalid-position-at-import-rule                no-invalid-position-at-import-rule
 1:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 1:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid
 1:1  ✖  Unknown rule no-irregular-whitespace                           no-irregular-whitespace

Reply:
I don't want to say it happens because of changes, but I don't understand a reason and ways to solve this